ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -137,7 +151,15 @@ func (request NetworkPacketLossRequest) ValidateRequest() error {
 	if request.Sources == nil || len(request.Sources) == 0 {
 		return fmt.Errorf(missingRequiredFieldError, "Sources")
 	}
-	return validateNetworkFaultRequestSources(request.Sources)
+	err := validateNetworkFaultRequestSources(request.Sources, "Sources")
```

Comment:
non-blocking - we can check the error like this
```
if err := someFunction(); err != nil {
    // handle error
}
```

Reply:
Addressing this in the next PR: https://github.com/aws/amazon-ecs-agent/pull/4373